doc/node-gyp.md

```diff
@@ -75,8 +75,8 @@ NODE_API_MODULE(NODE_GYP_MODULE_NAME, Init)
   - [Command options](https://www.npmjs.com/package/node-gyp#command-options)
   - [Configuration](https://www.npmjs.com/package/node-gyp#configuration)
 
-Sometimes finding the right settings for ```binding.gyp``` is not easy so to
-accomplish at most complicated task please refer to:
+Sometimes finding the right settings for `binding.gyp` is not easy, so to
+accomplish the most complicated tasks, please refer to:
 
 - [GYP documentation](https://gyp.gsrc.io/index.md)
 - [node-gyp wiki](https://github.com/nodejs/node-gyp/wiki)
```

Comment:
Could you update this link that is moved to `https://github.com/nodejs/node-gyp/tree/main/docs`?